CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+Thank you for considering to contribute and making custom on-demand SAR processing accessible!
```

Comment:
Wording feels awkward, maybe: Thank for you considering helping make custom on-demand SAR processing accessible!

Reply:
You're right. Hmm... maybe:

> Thank you for your interest in helping make custom on-demand SAR processing accessible!

Reply:
Perfect!